build/cloudbuild/cloudbuild.yaml

```diff
@@ -13,18 +13,10 @@
 # limitations under the License.
 
 steps:
-  - name: "gcr.io/cloud-builders/go:debian-1.18"
```

Comment:
Removed because the dockerfile already has a multistage build which downloads OCB